docs/developers/whitelisted-developermining.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Whitelisted Developer Mining Participants
```

Comment:
Related to this comment: https://github.com/UMAprotocol/docs/pull/149/files#r592449715

I think this should be deleted. It is really great content, but it would be best practice to have this all in one location. I think this should all be on the website instead. @Tommy1231232 do you think you have all of these on the website already?